source/connect.txt

```diff
@@ -21,12 +21,11 @@ Prerequisites
 .. include:: /includes/fact-connect-prereqs.rst
```

Comment:
[question] Does this include need an update? It says 4.0 or greater, but I'm not sure if we might be indicating what's possible here, rather than what's supported.